react/components/ExtensionPoint/ComponentLoader.tsx

```diff
@@ -129,15 +130,15 @@ const ComponentLoader: FunctionComponent<Props> = (props) => {
   )
 
   const shouldHydrate =
-    !hydration ||
+    hydration ||
```

Comment:
It seems weird.. 

`hydration || hydration === 'always'`.. any value of `hydration` will work here.

Reply:
It is weird! I will see with @lbebber if can just do `let shouldHydrate = hydration || ..`